Sources/PortalSwift/Connect/PortalConnect.swift

```diff
@@ -380,7 +380,11 @@ public class PortalConnect: EventBus {
         )
 
         let message = try JSONEncoder().encode(event)
-        self.client?.send(message)
+        
+        guard let client = self.client else {
+          throw PortalConnectError.noWebSocketClientFound
+        }
```

Comment:
maybe moving this to the beginning of the function would throw the error early before processing anything as it will fail anyways.

Reply:
Good call, will do!

Reply:
Updated ✅